tools/db_init.py

```diff
@@ -132,3 +139,5 @@ def test_db(database):
            {test_cmd + db}
         '''))
     sys.exit(1)
+
+print()
```

Comment:
what does this print statement do?

Reply:
It was just for some spacing in the log so this is more clearly delineated. Not super important, feel free to remove.